lib-ng/scanners/Beuckman.js

```diff
@@ -16,7 +17,10 @@ export default class Beuckman extends BaseScanner {
 
   startMonitoring(region) {
     if (this.Beuckman) {
-      this.Beuckman.startMonitoringForRegion(region);
+      const notifyEntryStateOnDisplay = sensimityConfig.notifyEntryStateOnDisplay || 'NO';
```

Comment:
Why not cast this variable? I'd expect boolean true as setting in the config. This is the Objective-C value exposed in Javascript.